src/test/handler.test.ts

```diff
@@ -1,25 +1,32 @@
+import { createStubSourcegraphAPI } from '@sourcegraph/extension-api-stubs'
 import expect from 'expect'
 import mock from 'mock-require'
-import { createMockSourcegraphAPI, projects } from './stubs'
+import { projects } from './extension.test'
 
-const sourcegraph = createMockSourcegraphAPI()
-// For modules importing Range/Location/URI/etc
+export const sourcegraph = createStubSourcegraphAPI()
+// For modules importing Range/Location/Position/URI/etc
 mock('sourcegraph', sourcegraph)
 
-import { checkMissingConfig, getParamsFromUriPath, matchSentryProject } from '../handler'
-import { SentryProject } from '../settings'
+import { checkMissingConfig, createDecoration, getParamsFromUriPath, matchSentryProject } from '../handler'
 
 describe('getParamsFromUriPath', () => {
+    beforeEach(async () => {
+        await sourcegraph.configuration.get().update('sentry.organization', 'sourcegraph')
+        await sourcegraph.configuration.get().update('sentry.projects', projects)
+    })
     it('extracts repo and file params from root folder', () =>
         expect(getParamsFromUriPath('git://github.com/sourcegraph/sourcegraph?264...#index.tsx')).toEqual({
-            repo: 'sourcegraph/sourcegraph',
+            repo: 'sourcegraph/sourcegraph?264...#index.tsx',
```

Comment:
Why is the file path included in the `repo` field? That doesn't look right

Reply:
I extended the repo path to include more of the file path because I realized that some files inside the same repo might be reporting to different Sentry projects. For example, `sourcegraph.+util/` reports to the sourcegraph-dot-com Sentry project, but `sourcegraph.+browser/` reports to the sourcegraph-browser Sentry project. In order for the user to setup a repo RegExp that will identify projects inside one repo to report to different Sentry projects, I need to have a more detailed repo path. I could optimize this by excluding the file at the end though to make it more specific.

Reply:
It is just very confusing that the property named "repo" contains more than just the repo, and the URI parse function now doesn't really parse much other than stripping the `git://github.com/` prefix. Why can't this be multiple properties?

Reply:
hm so additional to `repo`, also have sth like repoProject` to match against?